ludwig/data/cache/util.py

```diff
@@ -12,7 +12,7 @@ def calculate_checksum(original_dataset: CacheableDataset, config: ModelConfigDi
         "dataset_checksum": original_dataset.checksum,
         "global_preprocessing": config.get(PREPROCESSING, {}),
         "global_defaults": config.get(DEFAULTS, {}),
-        "feature_names": [feature[NAME] for feature in features],
+        "feature_proc_columns": [feature[PROC_COLUMN] for feature in features],
```

Comment:
Just to be safe for now, I'd suggest adding back `feature_names`. Just to be 100% sure we're net decreasing the number of false positive cache hits.

Reply:
Sounds good!